src/neo-asm-cli/readme.md

```diff
@@ -0,0 +1,39 @@
+# NEO-ASML
+
+NEO-ASML 是NEOVM的汇编语言
+
+他看起来像这样
+
+```
+Main()
+{
+    PUSH 1//push 1 number
+    PUSH 2
+    CALL method1
+    RET;
+}
+method1()
+{
+label1：
+    ADD
+88:
+    RET
+}
+```
+
+NEO-ASML 是NEO智能合约编译器的中间件，其它语言的代码可以先编译为NEO-ASML，再调用本模块编译为AVM
```

Comment:
In english is better for the community

Reply:
need help,my english is so worse.

Reply:
NEO-ASML is the middleware of the NEO smart contract compiler. The code of other languages can be compiled into NEO-ASML first, and then call this module to compile to AVM.